tensorrt_llm/_torch/auto_deploy/models/modeling_nemotron_h.py

```
@@ -0,0 +1,543 @@
+
+# Copied from transformers.models.mamba.modeling_mamba2.Mamba2PreTrainedModel
+class NemotronHPreTrainedModel(PreTrainedModel):
+    """
+    An abstract class to handle weights initialization and a simple interface for downloading and loading pretrained
+    models.
+    """
+
+    # NOTE: `config_class` is left out so as to not depend on the HF checkpoint's `configuration_nemotron_h.py`, nor
+    # have to copy it here.
+    base_model_prefix = "backbone"
+    _no_split_modules = ["NemotronHBlock"]
```

Ruff (0.14.5)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)